content/client-lib-development-guide/features.textile

```diff
@@ -544,7 +544,7 @@ h3(#realtime-channel). Channel
 ** @(RTL5h)@ If the connection state is @CONNECTING@ or @DISCONNECTED@, do the detach operation once the connection state is @CONNECTED@
 ** @(RTL5d)@ Otherwise a @DETACH@ ProtocolMessage is sent to the server, the state transitions to @DETACHING@ and the channel becomes @DETACHED@ when the confirmation @DETACHED@ ProtocolMessage is received
 ** @(RTL5f)@ Once a @DETACH@ @ProtocolMessage@ is sent, if a @DETACHED@ @ProtocolMessage@ is not received within the "default realtime request timeout":#defaults, the detach request should be treated as though it has failed and the channel will return to its previous state
-** @(RTL5k)@ If the channel receives an @ATTACHED@ message while in the @DETACHING@ state, it should send a new @DETACH@ message and remain in the @DETACHING@ state
+** @(RTL5k)@ If the channel receives an @ATTACHED@ message while in the @DETACHING@ or @DETACHED@ state, it should send a new @DETACH@ message and remain in the @DETACHING@ state
```

Comment:
> `remain in the @DETACHING@ state`

should probably now be 'remain (or transition) to'

Reply:
whoops, 31e2c7f.

Reply:
Resolving this conversation on @SimonWoolf's behalf so that we can merge this pull request cleanly, given it seems clear that @owenpearson has included the suggested change.